app/helpers/request_helper.py

```diff
@@ -11,6 +16,10 @@ def service_name(url=None):
             return 'SDX_STORE'
         elif 'sequence' in parts:
             return 'SDX_SEQUENCE'
+        elif 'cora' in parts:
+            return 'SDX_TRANSFORM_CORA'
+        else:
```

Comment:
Fair enough - it's totally optional, so no harm either way. Explicit is (mostly) better than implicit.